### PR TITLE
Fix securitron not dropping loot

### PR DESCRIPTION
**Pull request by Cenrus**
Securitrons now drop their uncharged baton when destroyed. Securitron noloot was set to true, while this variable is intended to prevent null crate beepsky robots from dropping eswords.

:cl:
fix: Beepsky and other securitron robots now drop their baton again
/:cl:

### CODE REVIEW

**Comment by VictorVow** [comment]
Was wondering why this was silently changed.

**Comment by ghost** [comment]
> Was wondering why this was silently changed.

This was changed with #1808 and is specifically stated in the changelog.

**Comment by ghost** [comment]
I don't think this is a good idea, the stun baton is one of the best if not the best melee weapon in the game and is rightfully restricted to Security, destroying a securitron is too easy to warrant this.

**Comment by Cenrus** [comment]
> 
> 
> I don't think this is a good idea, the stun baton is one of the best if not the best melee weapon in the game and is rightfully restricted to Security, destroying a securitron is too easy to warrant this.

Stupid change. Why do you insist that batons should be limited to a sec only weapon?

**Comment by ghost** [comment]
> Stupid change. Why do you insist that batons should be limited to a sec only weapon?

I just explained why above, read the sentence.

**Comment by Cenrus** [comment]
Right, let me rephrase it. Why do you think security should be the only ones allowed to use "one of the best if not the best melee weapon in the game"?

**Comment by ghost** [comment]
> Right, let me rephrase it. Why do you think security should be the only ones allowed to use "one of the best if not the best melee weapon in the game"?

They shouldn't, and I never said that either, it's the fact that it's way too easy to destroy a securitron and then get a 2 hit kill weapon from it.

**Review by PowerfulBacon — APPROVED** [review]
As long as the quad e-sword bots have noloot set to true

**Comment by eeSPee** [comment]
> > Right, let me rephrase it. Why do you think security should be the only ones allowed to use "one of the best if not the best melee weapon in the game"?
> 
> They shouldn't, and I never said that either, it's the fact that it's way too easy to destroy a securitron and then get a 2 hit kill weapon from it.

Why do we have 2hit kill weapons widely available roundstart and for free?

**Comment by PowerfulBacon** [comment]
Securitrons are pretty hard to destroy and if sec are going to spam a shit ton of 1 hit kill bots there should be a downside (plus this came from a zesko PR where it was quietly snuck in with a bunch of other changes).
Also been told that the quad e-sword robots have nodrop so all good.
